New documents added

Ashu104 opened the pull request:
## What does this PR do?  (changes made)

- add important document to cheatsheet repo
- some links are also updated

eshellman commented:
Don't invent titles! Use the title given by the resource.
Why have you removed two resources?
when you've made changes, please add `(changes made)` to the title of your PR

Ashu104 commented:
@eshellman actually I've updated the old link and added some new content. which are more efficient.
sure, I'll take care to add labels for future commits.
